src/pages/traits/list/index.tsx

```diff
@@ -0,0 +1,19 @@
+import TraitsListComponent from "@components/pages/traits/list";
+import { axGetTraitListData } from "@services/traits.service";
+import { getLanguageId } from "@utils/i18n";
+import React from "react";
+
+const traitsCreate = ({ data, filter }) => <TraitsListComponent data={data} filterKey={filter} />;
```

Comment:
for list page rename this `traitsCreate` to `traitsList` maybe to avoid confusion